tests/test_bot.py

```diff
@@ -1332,27 +1336,32 @@ def test_set_game_score_3(self, bot, chat_id):
     @flaky(3, 1)
     def test_set_game_score_4(self, bot, chat_id):
         # NOTE: numbering of methods assures proper order between test_set_game_scoreX methods
-        game_short_name = 'test_game'
-        game = bot.send_game(chat_id, game_short_name)
 
-        score = int(BASE_TIME) - HIGHSCORE_DELTA - 2
+        def func():
 
-        message = bot.set_game_score(
-            user_id=chat_id,
-            score=score,
-            chat_id=game.chat_id,
-            message_id=game.message_id,
-            force=True,
-        )
+            game_short_name = 'test_game'
+            game = bot.send_game(chat_id, game_short_name)
+
+            score = int(BASE_TIME) - HIGHSCORE_DELTA - 2
+
+            message = bot.set_game_score(
+                user_id=chat_id,
+                score=score,
+                chat_id=game.chat_id,
+                message_id=game.message_id,
+                force=True,
+            )
+
+            assert message.game.description == game.game.description
+            assert message.game.animation.file_id == game.game.animation.file_id
+            assert message.game.photo[0].file_size == game.game.photo[0].file_size
 
-        assert message.game.description == game.game.description
-        assert message.game.animation.file_id == game.game.animation.file_id
-        assert message.game.photo[0].file_size == game.game.photo[0].file_size
+            # For some reason the returned message doesn't contain the updated score. need to fetch
+            # the game again...
+            game2 = bot.send_game(chat_id, game_short_name)
+            assert str(score) in game2.game.text
 
-        # For some reason the returned message does not contain the updated score. need to fetch
-        # the game again...
-        game2 = bot.send_game(chat_id, game_short_name)
-        assert str(score) in game2.game.text
+        expect_bad_request(func, 'Bot_score_not_modified', 'This test can be flaky')
```

Comment:
I guess the more important reason here is that this also depends on the previous tests to run properly? (same above)

Reply:
Yea but I ran the tests individually and they passed. So not sure what that was about.

Reply:
mhkay. Still, "This test can be flaky" seems a bit strange as informations given that the test already has the `@flaky` decorator 😉 Would making it more flaky (e.g. `@flaky(5,1)` - 1 out of 5 runs) help?

Reply:
not sure, but I'll try by only keeping the flaky and testing...